src/app/dashboard-widgets/work-item-widget/work-item-widget.component.html

```diff
@@ -7,13 +7,13 @@ <h2 class="card-pf-title">
     </div>
     <div class="card-pf-body f8-card-body">
       <div class="f8-blank-slate-card" *ngIf="(myWorkItemsCount | async) === 0">
-        <h3>Create Work Items</h3>
+        <h3>View planner</h3>
         <p>
           Planner will help you to create different types of work-items, allows you to assign team members and iterations.
         </p>
         <div class="f8-blank-slate-main-action">
           <button id="spacehome-workitems-create-button" class="btn btn-primary btn-lg"
-                  [routerLink]="[{ outlets: { action: 'add-work-item' } }]">Create work item</button>
+                  [routerLink]="[contextPath | async, 'plan']">View planner</button>
```

Comment:
@catrobson do we have guidelines on casing for button labels like this?

![screenshot from 2018-05-02 14-02-11](https://user-images.githubusercontent.com/3787464/39540810-caf32640-4e11-11e8-8bc9-3aef9180519f.png)

Within the same view we have "View planner", "Add Codebase", "Add to space", and "Add to Space".

Reply:
Okay I've included the button. Er when you say use original label, did you mean it should still be labelled 'Create work item'?

Reply:
Right, keep the "Create work item" button and the icon in the header